src/test/java/calculator/CalculratorValidation.java

```diff
@@ -0,0 +1,62 @@
+package calculator;
+
+import calculator.separator.CustomSeparator;
+import org.thymeleaf.expression.Numbers;
+
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static calculator.separator.CustomSeparator.*;
+
+/**
+ * <pre>
+ * calculator
+ *      CalculratorValidation
+ * </pre>
+ *
+ * @author YunJin Choi(zzdd1558@gmail.com)
+ * @since 2022-03-02 오전 1:34
+ */
+
+public class CalculratorValidation {
+
+    public static boolean isNullOrEmpty(final String text) {
+        return isNull(text) || text.trim().isEmpty();
+    }
+
+    public static boolean isNull(final String text) {
```

Comment:
`CalculratorValidation` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private static boolean isNull(final String text) {
```